src/templates/app/new_account/virtual.jsx

```diff
@@ -25,10 +25,11 @@ const Virtual = () => (
                     </p>
                 </div>
             </Fieldset>
-            <div id='consent_checkbox'>
+            <div id='consent_checkbox' className='hide-product-checkbox'>
                 <FormRow
                     type='checkbox'
                     id='email_consent'
+                    className='hide-product-checkbox'
```

Comment:
Hey, I'm wondering, why exactly do we need to give the same classname to both parent and child elements? Can we remove one of these or use different selector for child element?

Reply:
Hello akmal, the classname is to display and hide the checkbox, but also for validation on the input. I changed the name of the class on the checkbox now to validate-hidden-product-checkbox. I also noticed that I can't directly parse a class into the FormRow component as I did here. It doesn't display in the DOM. I resolved this now through javascript.

Reply:
Could you give to one of these classes a more targeted name? Currently, both of them have `checkbox`, and I don't really see from the code whether we have two different checkboxes, or one of them refers to something else rather than the checkbox.

Reply:
I changed the class names. Let me know if they are better this way or not.